src/ocaml_protoc_plugin/yojson.ml

```diff
@@ -8,4 +8,19 @@ type t =
   | `Assoc of (string * t) list
   | `List of t list
   | `Tuple of t list
-  | `Variant of string * t option ]
\ No newline at end of file
+  | `Variant of string * t option ]
```

Comment:
Let's remove the first type definition and only keep the module `Safe` that you added. I missed that opportunity earlier to make things clearer. All the deriving functions really expect `Yojson.Safe.t`, so by naming it `Yojson.t` I made things unnecessarily complicated.

Reply:
done